src/renderer/features/operations/OperationsValidation/lib/add-proxy-rules.ts

```diff
@@ -74,4 +76,26 @@ export const AddProxyRules = {
   },
 };
 
-export const addProxyValidator = createTxValidator();
+export const addProxyValidator = createTxValidator<{ proxyNumber: number }>({
+  additionalBalanceRules: [
+    ({ route, getBalance, asset, api, proxyNumber }) => {
+      const initiator = accountService.findInitiator(route);
+      if (!initiator) return;
+
+      const chainId = api.genesisHash.toHex();
+
+      const balance = getBalance(initiator.accountId, chainId, asset.assetId);
+
+      if (nullable(balance)) return;
+
+      const deposit = proxyService.getProxyDeposit(api, '0', proxyNumber + 1);
```

Comment:
Same here. We should use existing deposit here instead of '0'. Overall, '0' can only be used if it is the very `first` proxy being created